### DISCUSSION TITLE
MQTT Details?

### DISCUSSION THREAD

**Discussion by mmcc-xx** [blakeblackshear/frigate#4170] [Ask A Question] [answered]
Hi All - I'm hoping to get some more information on when  various MQTT messages are published.

If a person is detected and snapshots are enabled, is frigate/<camera_name>/person/snapshot published once and only once for the duration of that "event"?

If a person is detected, is a frigate/events message published every time that person moves within the frame?

Thanks,
Clark

**Comment by NickM-27** [comment, marked as answer]
> If a person is detected and snapshots are enabled, is frigate/<camera_name>/person/snapshot published once and only once for the duration of that "event"?

The snapshot is updated from time to time based on score and some other factors

> If a person is detected, is a frigate/events message published every time that person moves within the frame?

No, it has a publish on start when the object is detected, publishes updates when the object enters or exits a zone, publishes updates once a minute (object is stationary), and published on end

**Reply by mmcc-xx** [reply]
Thanks for the info - that saves me some messing around.

"The snapshot is updated from time to time based on score and some other factors"

Is this documented anywhere?

**Reply by NickM-27** [reply]
I don't believe so
